content/en/database_monitoring/setup_sql_server/troubleshooting.md

```diff
@@ -62,22 +64,32 @@ Look in the error message for HResult error codes. Here are some known codes:
 
 `-2147217843` **“login failed for user”**: this means the agent succeeded in establishing a connection to the host but the login was rejected for some reason.
 
+`-2147467259` **“could not open database requested for login”**: this error appears either due to network issues or due to an unknown database.
+
+Please see the section below for tips on how to troubleshoot
+
+If neither step produces meaningful help, or the error code you are seeing is not listed, Datadog recommends to use the `MSOLEDBSQL` driver or the `Microsoft ODBC Driver for SQL Server`. Either of these options will produce more meaningful error messages, which should help troubleshooting why the connection is failing.
```

Comment:
I would move this to after the troubleshooting tips.

Reply:
It only applies to this section though, not the one below it. I'll reorganize the sections a bit

Reply:
OK, I refactored the order of things a bit so hopefully it makes more sense. Basically the invalid string attribute error is the same thing, but recent versions of the agent have made it so we dont even report this anymore. Its leftovers from older agent versions to even call it out, but i didnt want to remove it completely